src/workerd/api/trace.c++

```diff
@@ -276,11 +276,11 @@ jsg::Dict<jsg::ByteString, jsg::ByteString> TraceItem::FetchEventInfo::Request::
       //(name == "authorization"_kj) || // covered below
       (name == "cookie"_kj) ||
       (name == "set-cookie"_kj) ||
-      (strstr(name.cStr(), "auth") != nullptr) ||
-      (strstr(name.cStr(), "jwt") != nullptr) ||
-      (strstr(name.cStr(), "key") != nullptr) ||
-      (strstr(name.cStr(), "secret") != nullptr) ||
-      (strstr(name.cStr(), "token") != nullptr)
+      name.contains("auth") ||
```

Comment:
Would suffixing these string literals with `_kjc` avoid runtime calls to `strlen` going from a C-string to a `kj::StringPtr`?

Reply:
Done.

Reply:
Nit; looks like there's one more of these in `exception.c++`.

Reply:
`exception.c++` fixed.

Reply:
Thanks!

Reply:
FWIW `_kjc` produces a `kj::ConstString` whereas `_kj` produces `kj::StringPtr`. The latter would have worked here and feels slightly lighter although probably makes no real difference.

Reply:
Thanks. I posed the question as I didn't see such an initializer in string.h, but good to know.

Reply:
@ohodson Hmm, I may have misspoke... but the other thing is, I'd expect the compiler can elide `strlen()` when the parameter is known at compile time.